src/utils/request/onUnhandledRequest.ts

```diff
@@ -116,6 +116,17 @@ ${handlers.map((handler) => `  • ${handler.info.header}`).join('\n')}`
   return `Did you mean to request "${handlers[0].info.header}" instead?`
 }
 
+export function tryAndSilenceIfThrows<Payload>(
```

Comment:
Could you please move this utility function to `src/utils/internal`? That's the directory where we keep such internal helpers.